Zuva DocAI.postman_collection.json

```diff
@@ -428,10 +372,22 @@
 							"name": "OCR Request Created",
 							"originalRequest": {
 								"method": "POST",
-								"header": [],
+								"header": [
+									{
+										"key": "Content-Type",
+										"name": "Content-Type",
+										"value": "application/json",
+										"type": "text"
+									}
+								],
 								"body": {
 									"mode": "raw",
-									"raw": "{\n    \"file_ids\": [\n        \"{{file_id}}\"\n    ]\n}"
+									"raw": "{\n    \"file_ids\": [\n        \"c5e41av1qk1er7odm79g\"\n    ]\n}",
```

Comment:
Was this intentionally changed from a variable to hard-coded?

Reply:
@Jenarth If I understand correctly these are under `Examples`, rather than the `file_id` used as part of making an API call.

The examples shouldn't contain a variable: you should be able to view examples without needing to set environment/collection variables.

Correct me if I'm wrong. This would apply for the following couple of comments.

Reply:
Yup all 3 of these are `Examples` so I think it would be best to have it as an example of an actual file ID rather than a variable. Though I can change them all to variables, but either way there's no functional difference (because these example requests can't be executed in Postman)

Reply:
Showing the ID also helps the reader know what format the IDs are. We do use strings, but it still helps to see a visualization of a sample identifier.